Makefile

```diff
@@ -41,6 +41,16 @@ integration: build-cli-cross-platform integration-only
 integration-only:
 	PATH=$(PWD)/bin:${PATH} go test -v github.com/lacework/go-sdk/integration -timeout 30m
 
+integration-lql: build-cli-cross-platform integration-lql-only
+
+integration-lql-only:
+	PATH=$(PWD)/bin:${PATH} go test -v github.com/lacework/go-sdk/integration -run TestQuery* -timeout 30m
```

Comment:
Do you need the `*`? 🤔  Just curious since I never used it before.

Reply:
I will verify.  It appears that `-run` is a regexp:

```
-run regexp
        Run only those tests and examples matching the regular expression.
        For tests the regular expression is split into smaller ones by
        top-level '/', where each must match the corresponding part of a
        test's identifier.
```

Reply:
Works without `*`... Removing...